src/Core/src/Platform/Android/PickerManager.cs

```diff
@@ -40,12 +40,12 @@ public static void OnFocusChanged(bool gainFocus, EditText sender)
 
 		static void OnKeyPress(object? sender, AView.KeyEventArgs e)
 		{
+			//To prevent user from entering text when focus is received
+			e.Handled = true;
```

Comment:
I don't think we'll want to handle every single keypress event. 

This will probably break scenarios where a user has a hardware keyboard connected to the device and is trying to use tab to navigate components

Reply:
@PureWeen , Pressing the Tab key focuses on the EditText, causing the picker dialog to open. Navigation does not occur between the components. According to the implementation, a change in focus triggers the dialog to appear on the view. I have shared a video for reference. please let me know if you have any concerns.



https://github.com/user-attachments/assets/e284397b-738d-4994-b4bd-772993ebecea